.github/actions/setup-environment/action.yml

```diff
@@ -65,6 +65,9 @@ outputs:
   contentful-environment:
     description: 'Contentful Environment'
     value: ${{ steps.vars.outputs.contentful-environment }}
+  max-contentful-envs:
+    description: 'Maximum Contentful environments'
+    value: ${{ steps.vars.outputs.max-contentful-envs }}
```

Comment:
I don't mind this to be static like this but isn't there a way to do it via the contentful-checks action somehow?